### ISSUE TITLE
Accessibility : Drop Down Alert Error Message Not Communicated at same time it appear.

### ISSUE THREAD

**Issue by v-jashr** [OfficeDev/office-ui-fabric-react#11884]
![image](https://user-images.githubusercontent.com/50479173/73851597-c7cfef00-4853-11ea-80d1-068cdfe1b4c4.png)

when  Error Message appear it is not communicated to user. It read when we  go to another control and come back to it, then only it read.

Expected:
Alert should be communicated/given about error message at same time it appear.

**Comment by xugao** [comment]
@v-jashr - what type of screen reader + browser you having this issue with?
I can repro this with Narrator + Old Edge. However it's not reproing with Narrator + Edge Chromium nor NVDA + Chrome.
For the repro case, this appears to be a narrator issue. since old Edge is deprecating and it's working for the new Edge, I doubt they will make fix here

**Comment by v-jashr** [comment]
For Narrator and Edge it is not working for me. Could be please tell me which Edge Browser version and Narrator Combination work for you.

**Comment by xugao** [comment]
@v-jashr -
[Edge chromium stable version](https://www.microsoft.com/en-us/edge?form=MY01BL&OCID=MY01BL)
Version 79.0.309.71 (Official build) (64-bit)
